linkcheck/migrations/0005_url_add_fields.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations, models
+
+
+def reset_fields(apps, schema_editor):
+    """
+    Reset the status field of all Urls to indicate they need to be checked again to update the message
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+    # We can't import the Url model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    Url = apps.get_model("linkcheck", "Url")
+    Url.objects.all().update(status=None, last_checked=None, error_message="")
```

Comment:
Do you think it is ok to require a manual re-check after the update?
I thought re-checking could take a while, so probably users don't want this to happen during their migrations?